examples/yaml.rs

```diff
@@ -23,7 +23,8 @@ fn main() {
         }],
     };
 
-    let bytes = serde_yaml::to_vec(&person).unwrap();
```

Comment:
I don't think we need to change this code? Better keep it as it is to reduce risk IMHO. Or do we have good reason to change it? I seem to be able to build with serde_yaml 0.9 without this change.

Reply:
Sorry, my bad, I missed that this was just changing example code. I can reproduce the build problem when I build this example. Thanks for fixing.